src/app/public/modules/navbar/navbar-item.component.scss

```diff
@@ -8,6 +8,7 @@
 
   ::ng-deep {
     a,
+    .sky-dropdown-item > a,
```

Comment:
Why was this change necessary?

Reply:
Seems that a change in Dropdown's styles made it more difficult to overwrite the styles with a simple `a` selector. I had to add a more specific selector `.sky-dropdown-item > a` to overwrite the dropdown styles without using `!important`.

Reply:
Is this unique to the Angular 9 work? Or does this issue exist in the current master branch?

Reply:
It's happening in the master branch, yes. However, the fact that we haven't heard anything about this bug makes me think no one is using it. The template is using it, however, which is why we need to release it to support Angular 9.

At any rate, if you want to make this change to the `master` branch, too, here you go: https://github.com/blackbaud/skyux-navbar/pull/6

Reply:
Very good! Thanks for doing that. Ok, lets ship it.